DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -92,12 +90,6 @@ class GlobalAlgBlk
 
 private:
 
-    /// orbit number
-    int m_orbitNr;
-
-    /// bunch cross number of the actual bx
-    int m_bxNr;
-
```

Comment:
Right, the orbitNr and bxNr were originially put in as placeholds, because
we thought these would be in the record, but they are not, so we removed
them as part of a general clean up.



On Fri, Apr 22, 2016 at 6:30 PM, tmatsush <notifications@github.com> wrote:

> In DataFormats/L1TGlobal/interface/GlobalAlgBlk.h
> <https://github.com/cms-sw/cmssw/pull/14215#discussion_r60811553>:
>
> > @@ -92,12 +90,6 @@ class GlobalAlgBlk
> >
> >  private:
> >
> > -    /// orbit number
> > -    int m_orbitNr;
> > -
> > -    /// bunch cross number of the actual bx
> > -    int m_bxNr;
> > -
>
> uGT read-out block does not [0] and will not [1] contain orbit/bx number.
> [0]
> https://twiki.cern.ch/twiki/bin/viewauth/CMS/GlobalTriggerUpgrade-ReadoutSpec
> [1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/UGT_readout_proposal
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14215/files/7fc1c32245bf180c47169d08e00adfb0fdadbf08#r60811553>
>



-- 
----------------------------------------
B. Winer